### PR TITLE
Dev/branched/dfpla= 239 add date time to placement application

### PR DESCRIPTION
**Pull request by jacmurphy**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DFPL-239

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/PlacementController.java [hunk 1]
```diff
@@ -15,6 +15,7 @@
 import uk.gov.hmcts.reform.fpl.model.CaseData;
 import uk.gov.hmcts.reform.fpl.model.event.PlacementEventData;
 import uk.gov.hmcts.reform.fpl.service.PlacementService;
+import uk.gov.hmcts.reform.fpl.service.time.Time;
```
Comment (tomaszpowroznik) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/PlacementController.java:18]: not needed

File: service/src/main/java/uk/gov/hmcts/reform/fpl/model/Placement.java [hunk 2]
```diff
@@ -31,6 +32,9 @@
     @JsonProperty("placementConfidentialDocuments")
     private List<Element<PlacementConfidentialDocument>> confidentialDocuments;
 
+    @JsonProperty("placementUploadDateTime")
+    public LocalDateTime placementUploadDateTime;
```
Comment (tomaszpowroznik) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/model/Placement.java:36]: private

File: service/src/main/java/uk/gov/hmcts/reform/fpl/service/PlacementService.java [hunk 3]
```diff
@@ -126,6 +126,8 @@ public PlacementEventData savePlacement(CaseData caseData) {
 
         placement.setApplication(sealingService.sealDocument(applicationDocument));
 
+        placement.setPlacementUploadDateTime(time.now());
```
Comment (tomaszpowroznik) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/service/PlacementService.java:129]: missing test that covers this

Comment (jacmurphy) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/service/PlacementService.java:129]: added in service/src/test/java/uk/gov/hmcts/reform/fpl/service/PlacementServiceTest.java

**Review by tomaszpowroznik — APPROVED** [review]

**Pull request merged**